go.mod

```diff
@@ -1,4 +1,4 @@
-go 1.19
+go 1.20
```

Comment:
Could we bump go version in a separate PR? I believe this is not strictly needed for SDK 0.47.3, right?